examples/jsm/controls/OrbitControls.js

```diff
@@ -1418,7 +1448,8 @@ class OrbitControls extends EventDispatcher {
 		scope.domElement.addEventListener( 'contextmenu', onContextMenu );
 
 		scope.domElement.addEventListener( 'pointerdown', onPointerDown );
-		scope.domElement.addEventListener( 'pointercancel', onPointerUp );
+		scope.domElement.addEventListener( 'pointercancel', onPointerCancel );
+		scope.domElement.addEventListener( 'pointerout', onPointerOut );
```

Comment:
Are the changes to `OrbitControls` related to #27333?

Ideally, these are reverted before the PR is merged 😅 .

Reply:
oops :-)
I removed those erroneous changes from the PR